pkgs/development/ocaml-modules/morbig/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildDunePackage, fetchFromGitHub, ocamlPackages }:
+
+buildDunePackage rec {
+  pname = "morbig";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "colis-anr";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-fOBaJHHP/Imi9UDLflI52OdKDcmMxpl+NH3pfofmv/o=";
+  };
+
+  duneVersion = "3";
+
+  nativeBuildInputs = with ocamlPackages; [
+    menhir
+  ];
+
+  propagatedBuildInputs = with ocamlPackages; [
+    menhirLib
+    ppx_deriving_yojson
+    visitors
+    yojson
+  ];
```

Comment:
These packages should be taken individually as arguments instead of the `ocamlPackagse` set.

Reply:
I can open a PR to fix that. Do you have a pointer to something that explains the difference?

Reply:
Currently, `ocaml-ng.ocamlPackages_4_12.morbig` fails to build.

Reply:
cf https://github.com/NixOS/nixpkgs/pull/226131